hooks/post-command

```diff
@@ -72,14 +72,22 @@ get_codecov_uploader() {
   fi
 
   # One-time step
-  curl \
-      -fSs \
-      --retry 5 \
-      https://keybase.io/codecovsecurity/pgp_keys.asc \
+  if [[ "${BUILDKITE_PLUGIN_CODECOV_PGP_PUBLIC_KEY_URL:-false}" = false ]]; then
```

Comment:
```suggestion
  if [[ "${BUILDKITE_PLUGIN_CODECOV_PGP_PUBLIC_KEY_URL:-false}" = "false" ]]; then
```
To avoid confusion for the reader as false is highlighted as a built-in.

Reply:
Code changed a bit in the commit I pushed to raise issues: https://github.com/joscha/codecov-buildkite-plugin/pull/28/commits/f69e53c6009e1aedb346796f63387e65559fa35c